hash/apple2_cass.xml

```diff
@@ -1484,6 +1484,19 @@
 		</part>
 	</software>
 
+	<software name="mahjong">
+		<description>Shi-ri Mahjong</description>
```

Comment:
Is this an official title used in the software or its documentation?  I'd prefer using standard romanisation if it isn't.

Reply:
![Cassette](https://ia801507.us.archive.org/5/items/mahjongtsukumooriginalappleii/Mahjong%20%28Tsukumo%20Original%20Soft%29%20-%20Tape.jpg)
No, and my Japanese is terrible. If you know more correct romanisation, then please don't hesitate to tell me or change it yourself.